plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -261,7 +244,6 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 		tf, err = p.client.OpsV1alpha1().Traceflows().Create(ctx, tf, v1.CreateOptions{})
 		if err != nil {
 			log.Printf("Failed to create traceflow CRD \"%s\", err: %s", tfName, err)
-			return nil
```

Comment:
It doesn't seem right to remove it, so do below. Otherwise the log and the behavior would be weird.